args.go

```diff
@@ -147,48 +155,77 @@ func (a *Args) DelBytes(key []byte) {
 //
 // Multiple values for the same key may be added.
 func (a *Args) Add(key, value string) {
-	a.args = appendArg(a.args, key, value)
+	a.args = appendArg(a.args, key, value, ArgsHasValue)
 }
 
 // AddBytesK adds 'key=value' argument.
 //
 // Multiple values for the same key may be added.
 func (a *Args) AddBytesK(key []byte, value string) {
-	a.args = appendArg(a.args, b2s(key), value)
+	a.args = appendArg(a.args, b2s(key), value, ArgsHasValue)
 }
 
 // AddBytesV adds 'key=value' argument.
 //
 // Multiple values for the same key may be added.
 func (a *Args) AddBytesV(key string, value []byte) {
-	a.args = appendArg(a.args, key, b2s(value))
+	a.args = appendArg(a.args, key, b2s(value), ArgsHasValue)
 }
 
 // AddBytesKV adds 'key=value' argument.
 //
 // Multiple values for the same key may be added.
 func (a *Args) AddBytesKV(key, value []byte) {
-	a.args = appendArg(a.args, b2s(key), b2s(value))
+	a.args = appendArg(a.args, b2s(key), b2s(value), ArgsHasValue)
+}
+
+// AddNoValue adds only 'key' argument.
+//
+// Multiple values for the same key may be added.
+// Different with Add empty string, like key1=&key1=
+// Only key in argumemt, like key1&key1
```

Comment:
I think this would be a better comment:
```go
// AddNoValue adds only 'key' as argument without the '='.
//
// Multiple values for the same key may be added.
```